src/Views/Homepage.vala

```diff
@@ -184,19 +184,31 @@ namespace AppCenter {
         }
 
         private async void load_banners () {
+            unowned FlatpakBackend fp_client = FlatpakBackend.get_default ();
+            var packages_by_release_date = fp_client.get_packages_by_release_date ();
+
+            foreach (var package in packages_by_release_date) {
+                if (recently_updated_carousel.get_children ().length () >= NUM_PACKAGES_IN_CAROUSEL) {
+                    break;
+                }
+
+                if (package.state == AppCenterCore.Package.State.NOT_INSTALLED) {
```

Comment:
This is a bit more tricky since you might have a native package from a user installation and a system installation, so you need to loop over the packages in `package.origin_package` to get both and check they're _all_ not installed.